apps/prairielearn/src/migrations/20240307213407_course_requests__referral_source__add.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE course_requests
```

Comment:
Were we also supposed to add a `date` field here so we know when it was requested?

Reply:
Thank you for the reminder! That column actually already exists, I just have to display it:

https://github.com/PrairieLearn/PrairieLearn/blob/978914470b4c92d9d9ac49b69fe359b9e7abd9fe/database/tables/course_requests.pg#L4